### DISCUSSION TITLE
Using a local docker MySQL Database

### DISCUSSION THREAD

**Discussion by rosiejeays** [w00kster/mariokart-leaderboard#23] [General]
- Overcoming obstacle of constantly changing IP address during development
- As we haven't given the project consistent love in the past 4 or so months, there hasn't been a need to stand the live DB back up after the Linode free period ended.
- local docker DB means there doesn't have to be a sustained frequency in access 
- Ideally I'd like to use it for work relating to the "Create functionality to link UI to DB" Issue (#7)

**Comment by paulhansford** [comment]
### Benefits

- local development can speed up SDLC
- less dependency on remote/hosted services

### Downsides

- collective knowledge is not high on running/maintaining local containerised DB
- DB management

**Reply by rosiejeays** [reply]
I know not how to connect to the MySQL DB
